examples/gatsby/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load("@npm//gatsby-cli:index.bzl", "gatsby")
+
+gatsby(
+    name = "build",
+    args = ["build"],
+    data = [
+        ":src",
+        "@npm//:node_modules",
+    ],
+)
+
+filegroup(
+    name = "src",
```

Comment:
this rule comes first in the depgraph so I'd usually have it come first in the file